### ISSUE TITLE
Search is clipped on mobile 

### ISSUE THREAD

**Issue by Haroenv** [facebook/Docusaurus#385]
Hey there! Thanks for making docusaurus 👋 

### Is this a bug report?

Yes

### Have you read the [Contributing Guidelines](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md)?

Yes

### Steps to Reproduce

1. go to any of the websites using docusaurus
2. use Safari
3. open "responsive mode" (cmd-ctrl-r)
4. type something in the search bar

### Expected Behavior

The search results to take up most of the screen and have reasonable padding

### Actual Behavior

The search results are truncated and don't take up the whole screen. They also overlap a bit with the title. 

<img width="343" alt="screen shot 2018-01-09 at 11 37 21" src="https://user-images.githubusercontent.com/6270048/34717208-8ec08998-f532-11e7-94a6-36b05c03826e.png">

### Reproducible Demo

https://docusaurus.io has this for example

**Comment by JoelMarcey** [comment]
@Haroenv Thank you for this issue. Are you seeing this on Chrome too, btw?

**Comment by amyrlam** [comment]
I'm game to claim the issue! 

Seems like the styling of the search bar has been fixed as we replaced the site title with just an icon when on mobile. 

To clarify, it seems like we want the Algolia results to extend to a much larger height? Maybe if it's cut off at height x it can be cut off at the middle of a row as well so that it's clear it can scroll? I can play around with the design. Here's a gif of what I see right now:

![search](https://user-images.githubusercontent.com/1372946/40408791-8f8b0994-5e1d-11e8-92b2-a462ea9ae30e.gif)

I see this on Safari and Chrome.
